synapse/config/tls.py

```diff
@@ -428,11 +302,7 @@ def read_tls_certificate(self) -> crypto.X509:
         # Check if it is self-signed, and issue a warning if so.
         if cert.get_issuer() == cert.get_subject():
             warnings.warn(
-                (
-                    "Self-signed TLS certificates will not be accepted by Synapse 1.0. "
-                    "Please either provide a valid certificate, or use Synapse's ACME "
-                    "support to provision one."
-                )
+                "Self-signed TLS certificates will not be accepted by Synapse 1.0."
             )
```

Comment:
I think I'd get rid of all this. It was mostly there to warn people before 1.0 was released.